### DISCUSSION TITLE
Where is the downlaod

### DISCUSSION THREAD

**Discussion by JulianTechDL1** [TheUsefulLists/UsefulMods#243] [Q&A] [answered]
I cant find any relase or link to a download to a jar file.

**Comment by ItsukaHiro** [comment]
!?
The icon beside the name is the link to Modrinth, Curseforge and Github that you can download the jar file.

**Reply by JulianTechDL1** [reply]
![image](https://github.com/TheUsefulLists/UsefulMods/assets/113527854/cf6a237c-d0d9-4cf3-ab04-85fc711f9c81)

**Reply by JulianTechDL1** [reply]
where wdym

**Reply by ItsukaHiro** [reply, marked as answer]
Scroll down and choose you game version, based on what kind of mods you want (performance, bugfix or enhance)
